include/h2o/configurator.h

```diff
@@ -168,6 +168,13 @@ ssize_t h2o_configurator_get_one_of(h2o_configurator_command_t *cmd, yoml_t *nod
  */
 char *h2o_configurator_get_cmd_path(const char *cmd);
 
+/**
+ * lib/handler/configurator/util.c
+ */
+int h2o_on_config_header_2arg(h2o_configurator_command_t *cmd, h2o_configurator_context_t *ctx, int cmd_id, yoml_t *node, void *header_cmd_vector);
```

Comment:
Please create a typedef for `H2O_VECTOR(h2o_headers_command_t)` (e.g. `typedef H2O_VECTOR(h2o_headers_command_t) h2o_headers_commands_t`) and use that type, instead of using `void *header_cmd_vector`.